gdx/src/com/badlogic/gdx/scenes/scene2d/ui/HorizontalGroup.java

```diff
@@ -124,8 +124,8 @@ private void computeSize () {
 		}
 		prefHeight += padTop + padBottom;
 		if (round) {
-			prefWidth = Math.round(prefWidth);
-			prefHeight = Math.round(prefHeight);
+			prefWidth = (float)Math.ceil(prefWidth);
+			prefHeight = (float)Math.ceil(prefHeight);
```

Comment:
We should be able to use `MathUtils.ceil()` here, and `MathUtils.floor()` for flooring. The range where those start to be inaccurate is only for floats that are both very large and negative, I think less than -16384 . It probably doesn't matter, since UI layout code isn't called thousands of times per frame. Just an option to consider if some of this PR actually is called very often.